docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  elasticsearch-plugin-debug:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+      target: elasticsearch-plugin-debug
+    environment:
+      - discovery.type=single-node
+      # NO DEBUG
```

Comment:
DEBUG and NO DEBUG are the same here

Reply:
In contrast to NO DEBUG ES_JAVA_OPTS, DEBUG mod adds the `-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005` in order to attach a debugger to the remote JVM.

Reply:
Arf, sorry, bad newline formatting, I misread :)